apps/air-discount-scheme/api/src/app/modules/discount/discount.resolver.ts

```diff
@@ -85,11 +85,11 @@ export class DiscountResolver {
 
   @Roles(Role.ADMIN)
   @UseGuards(RolesGuard)
-  @Mutation(() => Discount)
+  @Mutation(() => [Discount], { nullable: true })
   createExplicitDiscountCode(
     @Context('dataSources') { backendApi },
     @Args('input', { type: () => CreateExplicitDiscountCodeInput }) input,
-  ): Promise<Discount> {
+  ): Promise<Array<Discount>> {
```

Comment:
This is not backwards compatible. So you need to be careful that this is not breaking anything. Otherwise you could have opted for a new query `createExplicitDiscountCodes` which returns arrays.

Reply:
This does not have to be backwards compatible as the only functions using this are also being changed